resources/templates/index.html

```diff
@@ -82,13 +91,21 @@ <h2>Disclaimer<h2>
   </div>
 </div>
 {% end %}
+{% end %}
 
 {% block page_header_scripts %}
 <script type="text/javascript">
+ const NETWORK = "{{ network.name }}";
+
  window.addEventListener("load", function() {
-   const checkboxes = document.querySelectorAll("input[type=checkbox]");
    const button = document.querySelector("button.link-button");
-   checkAcknowledgements(checkboxes, button);
+
+   if (NETWORK === "mainnet") {
+    const checkboxes = document.querySelectorAll("input[type=checkbox]");
+    checkAcknowledgements(checkboxes, button);
```

Comment:
I suspect this function is then calling `button.disable` as well? :thinking:  
I think the behavior between these two conditional cases is not too obvious. I would recommend to introduce to set the `disable` attribute of the button conditionally on the template. And then you still add this `checkAcknowledgements()` later without the `else` case. It would be also pretty cool when this function would work for mainnet and testnet, while doing nothing on testnet. I have no insights into this function (because I'm too lazy to search), so maybe this is not possible. I would imagine that `querySelectorAll` would return `undefined` on testnet (since the template does not render it) and then you simply do nothing. :shrug: 
Evaluate on your own if you like this idea and if you want to adjust for both or only the first suggestion. :upside_down_face:

Reply:
You are right. Your way doing it is prettier. I'll do both suggestions.